### PR TITLE
Enhancement: Bug Details Page - Task 2

### PR DESCRIPTION
**Pull request by CodeWithBishal**
Related #1627 - 2
created a layout for the page
created a sidebar for like,dislike,flag, and other btns
page is reponsive
files formatted using pre-commit run --all-files
todo add bookmark

Before
Screenshot 2024-03-09 at 4 49 58 PM

After
Screenshot 2024-03-09 at 6 37 48 PM
Screenshot 2024-03-09 at 6 37 59 PM

### CODE REVIEW

File: website/templates/issue3.html [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    {% include "includes/_likes2.html" %}
+    {% include "includes/_dislike2.html" %}
+    {% include "includes/_flags2.html" %}
+    {% include "includes/_bookmark2.html" %}
+    {% endcomment %}
+    {% include "includes/_like_dislike_share.html" %}
+    <div class="pt-8 pb-16 lg:pt-16 lg:pb-24 bg-white pl-[3%]">
+        <h2 class="font-[ubuntu] font-bold text-3xl">{{ object.description | truncatechars:100|escapejs }}</h2>
+        <p>
+            <a href="{{ object.url }}" class="text-red-600" target="_blank">{{ object.url | truncatechars:100 }}</a>
```
[the remark this answers is not in the document]
Comment (CodeWithBishal) [reply website/templates/issue3.html]: Fixed now :)

**Comment by CodeWithBishal** [comment]
@DonnieBLT  Okay I can get back /issue/ page working but changing the design here would mean I have to again start from scratch.

**Comment by CodeWithBishal** [comment]
@DonnieBLT yes the end goal is to replicate the same UI, I am just creating the layout now and incorporating some basic features

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
We can merge this when the previous issue URLs are working

**Review by DonnieBLT — APPROVED** [review]

**Comment by CodeWithBishal** [comment]
@DonnieBLT Sure,

Should I update the branch? there is only one merge conflict with base.html

**Pull request merged**